quiz_hosting_web_app/quiz_attempter_management/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from .models import Answer, Mark, Discussion, Comment
+
+
+@admin.register(Answer)
```

Comment:
Another way for registering models instead of using decorator.
```suggestion
admin.site.register(Answer, AnswerAdmin)
```

Reply:
I preferred this way because I find this one a bit easier